tests/TestCase/View/Helper/PaginatorHelperTest.php

```diff
@@ -1643,18 +1863,6 @@ public function testNumbersModulus()
         ];
         $this->assertHtml($expected, $result);
 
-        $result = $this->Paginator->numbers(['first' => 2, 'modulus' => 2, 'last' => 2]);
-        $expected = [
-            ['li' => []], ['a' => ['href' => '/index']], '1', '/a', '/li',
-            ['li' => []], ['a' => ['href' => '/index?page=2']], '2', '/a', '/li',
-            ['li' => ['class' => 'active']], '<a href=""', '3', '/a', '/li',
-            ['li' => []], ['a' => ['href' => '/index?page=4']], '4', '/a', '/li',
-            ['li' => ['class' => 'ellipsis']], '&hellip;', '/li',
-            ['li' => []], ['a' => ['href' => '/index?page=4896']], '4,896', '/a', '/li',
-            ['li' => []], ['a' => ['href' => '/index?page=4897']], '4,897', '/a', '/li',
-        ];
-        $this->assertHtml($expected, $result);
```

Comment:
These 11 lines are exactly same as previous 11 lines.

Previously first had no `'separator'` and second had `'separator' => ' - '`, but after removal of option they become duplicate.

See https://github.com/cakephp/cakephp/commit/3af7ef6c985a32cc088ede93afd4e3e02797e828#diff-63d45c463781a3bd23d79e9df334e52cR1416